### ISSUE TITLE
PDF created by merging multiple PDFs shows blank form fields when opened in Adobe Acrobat

### ISSUE THREAD

**Issue by jamoham** [py-pdf/pypdf#299]
When I create a PDF by merging multiple PDFs then only the first page shows up correctly but all subsequent pages show blank form fields. This happens only when this file is viewed on Adobe Acrobat.

## Code + PDF

Please refer the attachment and compare Page1 and Page2 of the PDF:
[document-output.pdf](https://github.com/mstamy2/PyPDF2/files/577060/document-output.pdf)

```python
from PyPDF2 import PdfMerger

merger = PdfMerger()
merger.append("working_us_report.pdf")
merger.append("working_us_report-2.pdf")

merger.write("document-output.pdf")
```

**Comment by Gunkkk** [comment]
Same problem!

**Comment by taka1426** [comment]
I'm having the same problem.
I like the "pypdf" wonderfully, but I think it will be even better once this issue is resolved.

### input file:
[1.pdf](https://github.com/py-pdf/pypdf/files/10317888/1.pdf)
[2.pdf](https://github.com/py-pdf/pypdf/files/10317889/2.pdf)

### output file:
[PyPDF2_out.pdf](https://github.com/py-pdf/pypdf/files/10317891/PyPDF2_out.pdf)

### code

import PyPDF2

merger = PyPDF2.PdfFileMerger()

merger.append('1.pdf')
merger.append('2.pdf')

merger.write('PyPDF2_out.pdf')
merger.close()

**Comment by pubpub-zz** [comment]
@taka1426 
can your confirm this issue still exists with the latest pypdf version

**Comment by pubpub-zz** [comment]
@taka1426,
this should be #fixed with #1553 (should be released this week-end). Can you retry it then? (do not forget to use `add_form_topname()` )

**Comment by pubpub-zz** [comment]
Without news I close this issue as solved
